crypto-onramp/src/main/java/com/stripe/android/crypto/onramp/ui/VerifyKycInfoActivity.kt

```diff
@@ -0,0 +1,88 @@
+package com.stripe.android.crypto.onramp.ui
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.os.Parcelable
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.activity.result.contract.ActivityResultContract
+import com.stripe.android.crypto.onramp.model.KycRetrieveResponse
+import com.stripe.android.crypto.onramp.model.RefreshKycInfo
+import com.stripe.android.link.LinkAppearance
+import com.stripe.android.paymentsheet.ui.KYCRefreshScreen
+import kotlinx.parcelize.Parcelize
+
+internal class VerifyKycInfoActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        val kycInfo = intent.getParcelableExtra<KycRetrieveResponse>("kycInfo")!!
+        val linkAppearance = intent.getParcelableExtra<LinkAppearance>("linkAppearance")
```

Comment:
1. avoid using these deprecated methods (see what we do elsewhere)
2. extract these key values to consts especially since they need to be in sync with `createIntent()`